WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Stats/DashboardStatsCardCell.swift

```diff
@@ -114,7 +114,7 @@ extension DashboardStatsCardCell: BlogDashboardCardConfigurable {
         WPAnalytics.track(.dashboardCardItemTapped,
                           properties: ["type": DashboardCard.todaysStats.rawValue],
                           blog: blog)
-        StatsViewController.show(for: blog, from: sourceController)
+        RootViewCoordinator.sharedPresenter.mySitesCoordinator.showStats(for: blog, source: .row, tab: .days, date: nil)
         WPAppAnalytics.track(.statsAccessed, withProperties: [WPAppAnalyticsKeyTabSource: "dashboard", WPAppAnalyticsKeyTapSource: "todays_stats_card"], with: blog)
```

Comment:
Now `stats_accessed` is tracked twice, once with `tap_source: row`, and another time with `tap_source: todays_stats_card`.

`WPAppAnalytics.track(.statsAccessed..` needs to be removed, and `showStats` needs to be called with a different source.

Reply:
Good catch! I forgot to remove this, thanks for fixing and merging 🙇